### ISSUE TITLE
Two ellipses (circles) intersection

### ISSUE THREAD

**Issue by nblthree** [bpmn-io/path-intersection#17]
__Describe the Bug__

Doesn't return the right result

__Steps to Reproduce__

use the following two paths
```
<path d="M2.6146209161795992e-14,73 A427,427 -90,0,0 -7.843862748538798e-14,927 A427,427 -90,1,0 2.6146209161795992e-14,73" stroke="black" stroke-width="10" fill="none"/>
<path d="M71.048,16.789855835444428 A439.5,439.5 0,0,1 928.6918008943089,15.63106872689174 A439.5,439.5 0,1,1 71.048,16.789855835444428" stroke="black" stroke-width="10" fill="none"/>
```
**A codesandbox** 

https://codesandbox.io/s/youthful-banzai-nd88h?file=/src/index.js

__Expected Behavior__

return the right intersections

__Environment__

- Host (Browser/Node version), if applicable: latest chrome, latest firefox
- OS: Windows 10
- Library version: 2.2.0

**Comment by nikku** [comment]
Thanks for the comprehensive bug report. I'll have a look, but cannot promise a quick fix.

**Comment by nikku** [comment]
Released `v2.2.1` with a fix.
